assets/sass/components/setup/_googlesitekit-setup.scss

```diff
@@ -127,14 +128,21 @@
 	margin-right: 16px;
 }
 
-.googlesitekit-wizard {
+.googlesitekit-setup__warning {
+	align-items: center;
+	display: flex;
```

Comment:
@asvinb are there `mdc-layout-*` classes we can use for these? 🤔

Reply:
@aaemnnosttv Using the `mdc-layout-*` classes are overkilled for this since the `mdc-layout-*` is using the grid layout. The grid gap is set to 0 on certain breakpoints, resulting in the overlapping icon and when the content for a cell is greater than the cell itself. For eg. the warning icon.